src/ReinforcementLearningEnvironments/src/environments/examples/TigerProblemEnv.jl

```diff
@@ -47,9 +47,7 @@ The main difference compared to other environments is that, now we have two
 kinds of *states*. The **observation** and the **internal state**. By default we
 return the **observation**.
 """
-RLBase.state(env::TigerProblemEnv) = state(env, Observation{Int}())
-
-function RLBase.state(env::TigerProblemEnv, ::Observation)
+function RLBase.state(env::TigerProblemEnv, ::Observation{Int}, ::DefaultPlayer)
```

Comment:
Isn't this breaking? Since `state(env)` no longer exists? In that case RLEnv needs a new version.